StandardSocketsHttpHandler/Net/Http/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -6,20 +6,14 @@ namespace System.Net.Http
 {
     internal static class HttpRequestMessageExtensions
     {
+        // depending on runtime the backing field for headers is either _headers or headers
+        // _headers in .net core and some .net fx and mono versions.
+        // headers in other .net fx / mono versions.

```

Comment:
Please be more descriptive and detail the info we already know ('.NET Framework 4.6.1', 'Mono on Mac')